hack/config.sh

```diff
@@ -1,8 +1,18 @@
-binaries="cmd/virt-controller cmd/virt-launcher cmd/virt-handler cmd/virt-api cmd/virtctl cmd/virt-manifest"
-docker_images="$binaries images/haproxy images/iscsi-demo-target-tgtd images/vm-killer images/libvirt-kubevirt images/spice-proxy cmd/virt-migrator"
-docker_prefix=kubevirt
-docker_tag=${DOCKER_TAG:-latest}
-manifest_templates="`ls manifests/*.in`"
-master_ip=192.168.200.2
-master_port=8184
-network_provider=weave
+unset binaries docker_images docker_prefix docker_tag manifest_templates \
```

Comment:
I think this will break the Vagrantfile. This file is also used as a config file:

```
$config = Hash[*File.read('hack/config.sh').split(/=|\n/)]

$master_ip = $config["master_ip"]
$network_provider = $config["network_provider"]
```

Maybe we can have a real ini file instead, which both can use to get their defaults?